index.html

```diff
@@ -1,8 +1,73 @@
 <!DOCTYPE html>
 <html>
    <head>
+      <title>Now Rosemarie Is Six</title>
+      <link rel="stylesheet" type="text/css" href="styles.css"/>
    </head>
    <body>
-      <!-- Write your HTML in this file! -->
+      <header>
+         <article>
+            <h1 id="mainHeading">Now We Are Six</h1>
+           <div class="extra-padding">
+            <p id= "testP" class="funParagraph" >My beautiful Rosemarie was born in April 2017. 
```

Comment:
Yay! You're the first GA4 I've graded who's hit enter within their paragraph tag for readability in the file. No infinite scrolling on one suuuuuper long line. Woohoo!